mod-orders-storage/schemas/receiving_history.json

```diff
@@ -59,7 +69,7 @@
   },
   "additionalProperties": false,
   "required": [
-    "caption",  
+    "caption",
     "poLineId",
     "receivingStatus",
     "supplement"
```

Comment:
should we make the purchaseOrderId also a required field since we will be searching on that field for showing it on UI?